kale/default_settings.py

```diff
@@ -34,6 +34,10 @@
 MESSAGE_QUEUE_PROXY_PORT = 0
 MESSAGE_QUEUE_PROXY_HOST = ''
 
+# Set to the VPC-compatible endpoint url to send SQS traffic through a VPC:
+# https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-sending-messages-from-vpc.html
+VPC_COMPATIBLE_ENDPOINT_URL = ''
```

Comment:
Can you remove `MESSAGE_QUEUE_USE_PROXY` as it is unused.
Also, I think it would be better to just change the `HOST/PORT` settings above to `MESSAGE_QUEUE_ENDPOINT_URL` and provide an example.

Reply:
It's still used as a Dev setting, I thought. I still see it in our settings file for Local Dev tasks

Reply:
The setting is not used in any code in this library, and is redundant to supplying a URL.